lib/build.gradle

```diff
@@ -42,6 +42,9 @@ dependencies {
     implementation "org.apache.arrow:arrow-memory-core:12.0.1"
     implementation "org.apache.arrow:arrow-vector:12.0.1"
 
+    implementation "com.fasterxml.jackson.core:jackson-annotations:2.15.1"
+    implementation "com.fasterxml.jackson.core:jackson-annotations:2.15.1"
```

Comment:
```suggestion
```

Reply:
The second dependency was supposed to be the `jackson-core` fixed [here](https://github.com/cloudquery/plugin-sdk-java/pull/82/commits/a8b12bd4b776b6e3cb1dd562770bac17ac80ab50)

Reply:
Woops deleted one by mistake, let me bring it back

Reply:
Actually we don't need annotations at the moment, correct?

Reply:
I did add them [here](https://github.com/cloudquery/plugin-sdk-java/blob/1d31031b08d4784b54dc2a589d9c3b76d3e4f312/lib/src/main/java/io/cloudquery/transformers/NameTransformer.java#L23) in a previous PR, so we do support them for custom JSON field names.

Reply:
Ah, cool. Let me add it back